Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2550,7 +2550,7 @@ def lhegensim2018ml(fragment,howMuch):
                          '--era':'Run3',
                          '--customise':'Configuration/DataProcessing/RecoTLR.customisePostEra_Run3'},dataReco])
 
-steps['RECODR3_reHLT']=merge([{'--conditions':'auto:run3_data_relval', '--hltProcess':'reHLT'},steps['RECODR3']])
+steps['RECODR3_reHLT']=merge([{'--conditions':'auto:run3_data_prompt', '--hltProcess':'reHLT'},steps['RECODR3']])
```

Comment:
if you are running a `reHLT` workflow with an HLT menu that needs to be synced with some specific  L1 menu, this might not be the correct choice. @cms-sw/hlt-l2 please comment.

Reply:
`run3_data_prompt_relval` will load the same thing as `run3_data_prompt` but with the fixed L1T menu. I recommend using that key.

Reply:
Sorry, I missed the comment by Marco. Yes, it's clear that the GT should be a `_relval` one.